lib/active_admin/resource_controller/streaming.rb

```diff
@@ -20,6 +20,7 @@ def index
       def stream_resource(&block)
         headers['X-Accel-Buffering'] = 'no'
         headers['Cache-Control'] = 'no-cache'
+        headers["Last-Modified"] = Time.zone.now.ctime.to_s
```

Comment:
Minor style suggestions that, I think, should be equivalent:

```suggestion
        headers['Last-Modified'] = Time.current.ctime
```

Reply:
Looking at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified neither are quite correct. The format is slightly wrong and it should be in UTC. `Time.now.httpdate`. I will fix that.